src/definitions/behaviors/form.js

```diff
@@ -1327,7 +1327,7 @@ $.fn.form = function(parameters) {
                 // cast to string avoiding encoding special values
                 value = (value === undefined || value === '' || value === null)
                     ? ''
-                    : (settings.shouldTrim) ? String(value + '').trim() : String(value + '')
+                    : (settings.shouldTrim && rule.shouldTrim !== false) || rule.shouldTrim ? String(value + '').trim() : String(value + '')
```

Comment:
Isn't the same as `settings.shouldTrim || rule.shouldTrim !== false`?

Reply:
No, there is one usecase there you may want to have a rule **not** trim althouh the overall form setting shouldTrim=true.
So if form:shouldTrim=true, your proposed solution will always be true, even if one explicisely defines: rule: shouldTrim:false